keadm/cmd/keadm/app/cmd/edge/join.go

```diff
@@ -232,6 +270,9 @@ func createEdgeConfigFiles(opt *common.JoinOptions) error {
 	}
 
 	edgeCoreConfig.Modules.EdgeHub.WebSocket.Server = opt.CloudCoreIPPort
+	// TODO: remove this after release 1.14
+	// this is for keeping backward compatibility
+	// don't save token in configuration edgecore.yaml
```

Comment:
Let's also add a `depracated` comment in token field, then rermove it in future release.

Reply:
if we run edgecore diretly, i think we can allow users to fill in token in edgecore.yaml, or users should write this token to /etc/kubeedge/bootstrap-edgecore.conf, this is not easy use

Reply:
If user run edgecore diretly, they should prepare the certs by themselfs. And token field in config file is not safe, we must remove it in future. cc @vincentgoat How do you think?

Reply:
Agree. The token should be constrained into the file /etc/kubeedge/bootstrap-edgecore.conf for temp use.